.github/workflows/bare-diffs.yml

```diff
@@ -2,12 +2,6 @@ name: Check Template Bare Minimum Diffs
 
 on:
   workflow_dispatch: {}
-  push:
-    branches: [main]
-    paths:
-      - 'templates/expo-template-bare-minimum/**'
-      - '.github/workflows/bare-diffs.yml'
-      - 'docs/public/static/diffs/template-bare-minimum/**'
   pull_request:
```

Comment:
would be good to disable from pull_request as well. template changes from time to time, the meaningful time to generate the diff is before sdk releasing. maybe we can run it manually or only check on `sdk-*` branches. that would save time, making pr review easier and also keep git history size smaller.